core/templates/dev/head/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.controller.ts

```diff
@@ -121,71 +121,6 @@ angular.module('oppia').directive('topicsAndSkillsDashboardPage', [
             );
           };
 
-          ctrl.createTopic = function() {
```

Comment:
These 2 functions seem to have been duplicated in this controller, hence deleted. the duplicated instance.